### PR TITLE
Remove cuda 9 related packages from tests, update cupy to 7.5

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It updates test python packages

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     - removes cuda 9 related packages from tests
     - updates cupy to 7.5
 - Affected modules and functionalities:
     setup_packages.py
 - Key points relevant for the review:
     NA
 - Validation and testing:
     CI
 - Documentation (including examples):
     NA

**JIRA TASK**: *[NA]*

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: qa/setup_packages.py [hunk 1]
```diff
@@ -316,29 +316,24 @@ def get_pyvers_name(self, url, cuda_version):
 
 all_packages = [PlainPackage("opencv-python", ["4.2.0.32"]),
                 CudaPackage("cupy",
-                        { "90"  : ["7.3.0"],
-                          "100" : ["7.3.0"] },
+                        { "100" : ["7.5.0"] },
                         "cupy-cuda{cuda_v}"),
                 CudaPackage("mxnet",
                         { "90"  : ["1.6.0"],
```
Comment (awolant) [inline qa/setup_packages.py:322]: Probably this also should be removed here, right?

Comment (JanuszL) [reply qa/setup_packages.py]: Missed that. Done

**Comment by JanuszL** [comment]
!build

**Pull request merged**